templates/actor/legend-points/history-earned.hbs

```diff
@@ -1,134 +1,51 @@
-<form class="legend-point__history--earned flexcol" id="legend-point__history--earned-form" autocomplete="off">
+<div class="tab earned-tab" data-group="primary" data-tab="earned-tab">
+  <!-- Body -->
+  <div class="lp-history__earned-body">
+    <label>{{localize "X-Total LP"}}</label>
+    <input class="input__number-input--wide" name="this.object.system.lp.total" type="text" value="{{this.object.system.lp.total}}" data-dtype="String" />
+    <table class="input--centered">
+      <thead>
+        <tr>
+          <th>
+            {{localize "X-Date"}}
+          </th>
+          <th>
+            {{localize "X-Description"}}
+          </th>
+          <th>
+            {{localize "X-LP"}}
+          </th>
+          <th>
+            {{localize "X-Editors"}}
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each this.object.system.lp.earnings as | this key | }}
```

Comment:
for future use: the `as | value key |` really only is necessary if you want custom names the variables. Otherwise, you can call them with `this` and `@key` anyway. So it's not really necessary here.